tidb/datadog_checks/tidb/data/conf.yaml.example

```diff
@@ -50,46 +50,115 @@ instances:
     #
   - pd_metric_url: http://localhost:2379/metrics
 
+    ## @param pd_customized_metrics - list of strings - optional
+    ## List of metrics to be fetched from the prometheus endpoint in addition to default PD metrcis, if there's a
+    ## value it'll be renamed. This list should contain at least one metric. To collect all PD metrics, add `pd*`.
+    #
+    # pd_customized_metrics:
+    #   - processor:cpu
+    #   - memory:mem
+    #   - process_resident_memory_bytes: go_runtime.process_resident_memory.bytes
+
     ## @param tidb_metric_url - string - required
     ## The URL where TiDB metrics are exposed by Prometheus.
     #
     tidb_metric_url: http://localhost:10080/metrics
 
+    ## @param tidb_customized_metrics - list of strings - optional
+    ## List of metrics to be fetched from the prometheus endpoint in addition to default TiDB metrcis. If there's a
```

Comment:
Two things for this file's notes:

- Fix `metrcis` misspellings.
- Capitalize Prometheus.